f8a_worker/defaults.py

```diff
@@ -155,5 +155,26 @@ def select_random_github_token(cls):
             raise F8AConfigurationException("Limit for unauthorized GitHub access exceeded.")
         return token, headers
 
+    @classmethod
+    def libraries_io_project_url(cls, ecosystem, name):
+        """Construct url to endpoint, which gets information about a project and it's versions."""
+        url = '{api}/{platform}/{name}'. \
+            format(api=cls.LIBRARIES_IO_API,
+                   platform=ecosystem,
+                   name=name)
+
+        if not cls.LIBRARIES_IO_TOKEN or cls.LIBRARIES_IO_TOKEN == 'not-set':
```

Comment:
What is the difference between `not-set` and and `no-token`?

Reply:
`not-set` is default value, see
https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/defaults.py#L58
https://github.com/fabric8-analytics/fabric8-analytics-deployment/blob/master/openshift/env-template.sh#L33
It means that user has not set the `LIBRARIES_IO_TOKEN` and we want to explicitly tell the user that it is required to do so.

`no-token` is special value. By setting `LIBRARIES_IO_TOKEN` to `no-token` the user requests that (s)he explicitly does not want to use ANY token, see the comment in code below. We use this in tests where we don't want to store the token anywhere, but still be able to run (and pass) the tests locally.
I understand that `not-set` and `no-token` sounds similar, so if you have any suggestions, how to call it, fire.
Maybe `unauthenticated` ?

Reply:
No, it's OK. I just wanted to be sure whether it's not mistaken. Thanks!